TextMining.py

```diff
@@ -0,0 +1,163 @@
+"""
+John Wen - Text Mining assignment
+
+"""
+
+import requests
+import random
+
+def gettext(url):
+    """ maybe modify this to incorporate other websites
+        if I have the time"""
+    return requests.get(url).text
+
+Christmas = gettext('http://www.gutenberg.org/cache/epub/46/pg46.txt')
+#OliverTwist = gettext('http://www.gutenberg.org/ebooks/730.txt.utf-8')
+#DavidCopperfield = gettext('http://www.gutenberg.org/files/766/766-0.txt')
+#GreatExpectations = gettext('http://www.gutenberg.org/files/1400/1400-0.txt')
+#ATaleofTwoCities = gettext('http://www.gutenberg.org/files/98/98-0.txt')
+
+def cleanuplist(textlist):
+    """ takes a text as a string and returns a list of words
+        without any of the symbols and lowercased
+
+    >>> cleanuplist('This project is so hard!')
+    ['this', 'project', 'is', 'so', 'hard']
+    >>> cleanuplist('I need, a bunch, of !? doctest?')
+    ['i', 'need', 'a', 'bunch', 'of', 'doctest']
+
+    """
+    cleanedlist = []
+    textlist = textlist.lower().split()
+    for word in textlist:
+        symbols = "-_=+[}{]:;?/.>,<?!@#$%^&*()|'"

```

Comment:
This is a good way of implementing this function. I also suggest you look into regular expression if you want your code to be more concise! https://en.wikipedia.org/wiki/Regular_expression